backend/src/services/secret-v2-bridge/secret-v2-bridge-dal.ts

```diff
@@ -13,6 +20,97 @@ export type TSecretV2BridgeDALFactory = ReturnType<typeof secretV2BridgeDALFacto
 export const secretV2BridgeDALFactory = (db: TDbClient) => {
   const secretOrm = ormify(db, TableName.SecretV2);
 
+  const findOne = async (filter: Partial<TSecretsV2>, tx?: Knex) => {
+    try {
+      const docs = await (tx || db)(TableName.SecretV2)
+        .where(filter)
+        .leftJoin(
+          TableName.SecretV2JnTag,
+          `${TableName.SecretV2}.id`,
+          `${TableName.SecretV2JnTag}.${TableName.SecretV2}Id`
+        )
+        .leftJoin(
+          TableName.SecretTag,
+          `${TableName.SecretV2JnTag}.${TableName.SecretTag}Id`,
+          `${TableName.SecretTag}.id`
+        )
+        .select(selectAllTableCols(TableName.SecretV2))
+        .select(db.ref("id").withSchema(TableName.SecretTag).as("tagId"))
+        .select(db.ref("color").withSchema(TableName.SecretTag).as("tagColor"))
+        .select(db.ref("slug").withSchema(TableName.SecretTag).as("tagSlug"));
+
+      const data = sqlNestRelationships({
+        data: docs,
+        key: "id",
+        parentMapper: (el) => ({ _id: el.id, ...SecretsV2Schema.parse(el) }),
+        childrenMapper: [
+          {
+            key: "tagId",
+            label: "tags" as const,
+            mapper: ({ tagId: id, tagColor: color, tagSlug: slug }) => ({
+              id,
+              color,
+              slug,
+              name: slug
+            })
+          }
+        ]
+      });
+      return data?.[0];
+    } catch (error) {
+      throw new DatabaseError({ error, name: `${TableName.SecretV2}: FindOne` });
+    }
+  };
+
+  const find = async (filter: TFindFilter<TSecretsV2>, { offset, limit, sort, tx }: TFindOpt<TSecretsV2> = {}) => {
```

Comment:
When I try to move a secret from the main page I get this error pertaining to here (I'm guessing the same would apply to `findOne` above  (column reference "id" is ambiguous)

```
[01:46:05.094] ERROR (25674): Failed to execute db ops {"reqId":"req-g","severity":"ERROR"}
infisical-dev-api              |     err: {
infisical-dev-api              |       "type": "DatabaseError",
infisical-dev-api              |       "message": "Failed to execute db ops",
infisical-dev-api              |       "stack":
infisical-dev-api              |           secrets_v2: Find: Failed to execute db ops
infisical-dev-api              |               at Object.find (/app/src/services/secret-v2-bridge/secret-v2-bridge-dal.ts:2:3865)
infisical-dev-api              |               at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
infisical-dev-api              |               at async Object.moveSecrets (/app/src/services/secret-v2-bridge/secret-v2-bridge-service.ts:2:39697)
infisical-dev-api              |               at async Object.handler (/app/src/server/routes/v3/secret-router.ts:4:23736)
infisical-dev-api              |       "name": "secrets_v2: Find",
infisical-dev-api              |       "error": {
infisical-dev-api              |         "type": "DatabaseError",
infisical-dev-api              |         "message": "select \"secrets_v2\".*, \"secret_tags\".\"id\" as \"tagId\", \"secret_tags\".\"color\" as \"tagColor\", \"secret_tags\".\"slug\" as \"tagSlug\" from \"secrets_v2\" left join \"secret_v2_tag_junction\" on \"secrets_v2\".\"id\" = \"secret_v2_tag_junction\".\"secrets_v2Id\" left join \"secret_tags\" on \"secret_v2_tag_junction\".\"secret_tagsId\" = \"secret_tags\".\"id\" where (\"type\" = $1 and \"id\" in ($2)) - column reference \"id\" is ambiguous",
infisical-dev-api              |         "stack":
infisical-dev-api              |             error: select "secrets_v2".*, "secret_tags"."id" as "tagId", "secret_tags"."color" as "tagColor", "secret_tags"."slug" as "tagSlug" from "secrets_v2" left join "secret_v2_tag_junction" on "secrets_v2"."id" = "secret_v2_tag_junction"."secrets_v2Id" left join "secret_tags" on "secret_v2_tag_junction"."secret_tagsId" = "secret_tags"."id" where ("type" = $1 and "id" in ($2)) - column reference "id" is ambiguous
infisical-dev-api              |                 at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
infisical-dev-api              |                 at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
infisical-dev-api              |                 at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
infisical-dev-api              |                 at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
infisical-dev-api              |                 at Socket.emit (node:events:519:28)
infisical-dev-api              |                 at Socket.emit (node:domain:488:12)
infisical-dev-api              |                 at addChunk (node:internal/streams/readable:559:12)
infisical-dev-api              |                 at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
infisical-dev-api              |                 at Readable.push (node:internal/streams/readable:390:5)
infisical-dev-api              |                 at TCP.onStreamRead (node:internal/stream_base_commons:191:23)
infisical-dev-api              |         "length": 108,
infisical-dev-api              |         "name": "error",
infisical-dev-api              |         "severity": "ERROR",
infisical-dev-api              |         "code": "42702",
infisical-dev-api              |         "position": "354",
infisical-dev-api              |         "file": "parse_relation.c",
infisical-dev-api              |         "line": "805",
infisical-dev-api              |         "routine": "scanRTEForColumn"
infisical-dev-api              |       }
infisical-dev-api              |     }
```

Reply:
Resolved